README.md

```diff
@@ -13,32 +13,60 @@ Describe your example and specify what it is demonstrating. Consider the
 following questions:
 
 * Is it pedagogical or a usable application?
-* Does it belong to a particular domain such as material simulation or logistics? 
-* What level of Ocean proficiency does it target (beginner, advanced, pro)? 
+* Does it belong to a particular domain such as material simulation or logistics?
+* What level of Ocean proficiency does it target (beginner, advanced, professional)?
 
 A clear description allows us to properly categorize your example.
 
 Images are encouraged. If your example produces a visualization, consider
 displaying it here.
 
-![D-Wave Logo](dwave_logo.png)
+![D-Wave Logo](static/dwave_logo.svg)
 
-## Usage
+## Installation
+You can run this example without installation in cloud-based IDEs that support the [Development Containers specification](https://containers.dev/supporting) (aka "devcontainers").
 
-A simple command that runs your program. For example,
+For development environments that do not support `devcontainers`, install requirements:
 
 ```bash
-python <demo_name>.py
+pip install -r requirements.txt
 ```
 
+If you are cloning the repo to your local system, working in a [virtual environment](https://docs.python.org/3/library/venv.html) is recommended.
+
+## Usage
+Your development environment should be configured to [access Leap’s Solvers](https://docs.ocean.dwavesys.com/en/stable/overview/sapi.html). You can see information about supported IDEs and authorizing access to your Leap account [here](https://docs.dwavesys.com/docs/latest/doc_leap_dev_env.html).
+
 ### Inputs
 If your example requires user input, make sure to specify the input format and any input limitations.
 ### Outputs
 An example program output.
 
+## Demo Types
+### User Interface Demo (Dash Framework Template)
```

Comment:
Might be clearer to simply have "### Demos Using Dash Interface"

Presumably a user contributing to our examples will know we use Dash for fancy GUIs so this will be easily understood